app/lib/pre_assembly/digital_object.rb

```diff
@@ -89,7 +90,7 @@ def pre_assemble
       @assembly_directory = AssemblyDirectory.create(druid_id: druid.id, base_path: container)
       stage_files
       update_structural_metadata
-      StartAccession.run(druid: druid.druid, user: batch.batch_context.user.sunet_id)
+      StartAccession.run(druid: druid.druid, user: batch.batch_context.user.sunet_id, workflow: default_workflow)
       log "    - pre_assemble(#{druid.id}) finished"
```

Comment:
previously we would always get the default workflow started in DSA, which is `assemblyWF` ... I think we now want this to depend on the content type supplied